.github/workflows/buid_and_release.yml

```diff
@@ -16,5 +16,20 @@ jobs:
     - uses: actions/checkout@v2
     - name: build
       run: make all
+    - name: Get current time
+      uses: josStorer/get-current-time@v2.0.2
+      id: current-time
+      with:
+        format: YYYY-MM-DD
     - name: Release
-      run: make deploy-release
+      run: echo Uploading files as new release.
+    - uses: "marvinpinto/action-automatic-releases@latest"
+      with:
+        repo_token: "${{ secrets.GITHUB_TOKEN }}"
+        automatic_release_tag: "${{ steps.current-time.outputs.formattedTime }}"
+        title: "${{ steps.current-time.outputs.formattedTime }}"
+        prerelease: false
+        files: |
+          release/medgen.obo
```

Comment:
@matentzn I included these files in the release:
- `medgen.obo`
- `medgen-disease-extract.obo`
- `medgen-disease-extract.owl`

Lemme know if you want me to release all 3 of these or a subset.

If you want, I can also remove `medgen.obo` and rename `medgen-disease-extract` to just `medgen`. That'll be the one we're ingesting. But I kind of like the specificity in the file name.

Reply:
Its good, leave it as is!